lib/knife-acl/version.rb

```diff
@@ -1,3 +1,3 @@
 module KnifeACL
-  VERSION = "0.0.12"
+  VERSION = "0.0.13"
```

Comment:
We'll need to make this a more significant version change since we are changing the CLI.